CHANGELOG.md

```diff
@@ -2,11 +2,27 @@
 ====================
 
 ## Features
+
 ### Platform Specific Features
 
 #### Android
 - Added support for Android 34
 
+## Changes
+
+### Call Message Events (Beta)
```

Comment:
Here's my suggestion. Feel free to change the wording. But should have the same thought.

### Call Message Events

The Call Message Events feature, originally released in 1.0.0, has been promoted to GA. This release includes the following changes:

- **(Breaking)** Removed `CallMessage.MessageType` and `CallMessage.ContentType` enumerations and types.
  Instead, those types have been replaced by `string`. The following is an example of the updated API:
  ```ts
  const call = await voice.connect(...);
  call.sendMessage(new CallMessage({
    content: { foo: 'bar' },
    contentType: 'application/json',
    messageType: 'user-defined-message',
  }));
  ```